TINY-10237: Enable SVG support (#9046)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Colorpicker now includes the Brightness/Saturation selector and hue slider in the keyboard navigable items. #TINY-9287
- Improved the tooltips of picker buttons for the urlinput components in the "Insert/Edit Image" and "Insert/Edit Link" dialogs. #TINY-10155
- Inline dialog will now respect `size: 'large'` argument in the dialog spec. #TINY-10209

> - SVG elements and it's children is now retained if you configure in SVG elements as a valid element. #TINY-10237

Reviewer: SVG elements and their children are now retained when configured as valid elements.

> - SVG elements and it's children is now retained if you configure in SVG elements as a valid element. #TINY-10237

Reviewer: ```suggestion
- SVG elements and their children are now retained when configured as valid elements. #TINY-10237
```

Author: Yes that is much better than what I had. Thanks
